sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -423,6 +423,10 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
     props.putAll(connectionProperties)
     val conn = JdbcUtils.createConnectionFactory(url, props)()
 
+    // to add required options like URL and dbtable
+    val params = extraOptions.toMap ++ Map("url" -> url, "dbtable" -> table)
+    val jdbcOptions = new JDBCOptions(params)
+
     try {
       var tableExists = JdbcUtils.tableExists(conn, url, table)
```

Comment:
Maybe I think we can use `jsonOptions.url` and `jsonOptions.table` here. I know there are no differences but as we made the `JDBCOptions` object to tie up the extra options.